presto-docs/src/main/sphinx/language/types.rst

```diff
@@ -214,11 +214,21 @@ Structural
 ``ROW``
 ^^^^^^^
 
-    A structure made up of named fields. The fields may be of any SQL type, and are
-    accessed with field reference operator ``.``
+    A structure made up of fields that allows mixed types.
+    The fields may be of any SQL type.
+
+    By default, row fields are not named, but names can be assigned.
 
     Example: ``CAST(ROW(1, 2.0) AS ROW(x BIGINT, y DOUBLE))``
 
+    Named row fields are accessed with field reference operator ``.``.
+
+    Example: ``CAST(ROW(1, 2.0) AS ROW(x BIGINT, y DOUBLE)).x``
+
+    Named or unnamed row fields are accessed by position with subscript operator ``[]``.
```

Comment:
Missing "the" in "with the subscript operator". How about
```
Named or unnamed row fields are accessed by position with the subscript operator ``[]``.
The position starts at ``1`` and must be a constant.

Reply:
OK, I'll do it this way.
However, this section generally doesn't discuss field access at all. Row was the exception, because it hasn't got its own section in `7. Functions and Operators` - an so I followed and added more info here. Moving row operators to `7` could be an alternative if `Row` description ceases to be concise.